From 86f530bfa7902b8e35807537641d7c9344ea4246 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Tue, 15 May 2018 08:22:42 +0800
Subject: [PATCH 3804/5242] MLK-18311 ARM64: dts: imx8qxp: correct the rpmsg
 address on arm2 board

commit  b1d9fc574eb6326f17a811134db046beacaeddb4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Regarding to the latest layout of the reserved memory
on imx8qxp, correct the rpmsg address on arm2 board too.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index d52c3be..c3fcc7b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -715,10 +715,9 @@
 &rpmsg{
 	/*
 	 * 64K for one rpmsg instance:
-	 * --0xb8000000~0xb800ffff: pingpong
 	 */
 	vdev-nums = <1>;
-	reg = <0x0 0xb8000000 0x0 0x10000>;
+	reg = <0x0 0x90000000 0x0 0x10000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

